# Douglas Scofield UPPMAX 02/2026
# - nose 1.3.7 is python2-ish
# - this just removes nose requirement and nosetests
#################################
diff -ruN CONCOCT-1.1.0.orig/Makefile CONCOCT-1.1.0/Makefile
--- CONCOCT-1.1.0.orig/Makefile	2019-08-02 10:03:28.000000000 +0200
+++ CONCOCT-1.1.0/Makefile	2026-02-20 17:31:45.607410371 +0100
@@ -25,8 +25,8 @@
 lint:
 	flake8 concoct bin tests
 
-test:
-	nosetests
+#test:
+#	nosetests
 
 docs:
 	$(MAKE) -C doc clean
diff -ruN CONCOCT-1.1.0.orig/requirements.txt CONCOCT-1.1.0/requirements.txt
--- CONCOCT-1.1.0.orig/requirements.txt	2019-08-02 10:03:28.000000000 +0200
+++ CONCOCT-1.1.0/requirements.txt	2026-02-20 17:31:57.055357171 +0100
@@ -3,7 +3,6 @@
 bcbio-gff>=0.4; python_version >= '3.0'
 biopython>=1.63,<=1.72; python_version < '3.0'
 biopython>=1.63; python_version >= '3.0'
-nose>=1.3.0
 numpy>=1.8.0
 pandas>=0.13.0
 python-dateutil>=2.2
diff -ruN CONCOCT-1.1.0.orig/setup.py CONCOCT-1.1.0/setup.py
--- CONCOCT-1.1.0.orig/setup.py	2019-08-02 10:03:28.000000000 +0200
+++ CONCOCT-1.1.0/setup.py	2026-02-20 17:32:12.685283371 +0100
@@ -51,8 +51,7 @@
                         'scipy>=0.12.0',
                         'pandas>=0.11.0',
                         'biopython>=1.62b',
-                        'scikit-learn>=0.13.1',
-                        'nose'],
+                        'scikit-learn>=0.13.1'],
       entry_points="""
       # -*- Entry points: -*-
       """,
